Grammar, Mechanics, and Word List.md

```diff
@@ -1,2 +1,43 @@
+## Mechanics
+
+### Headlines
+
+For headline levels 1 and 2 (H1 and H2) we follow general title case guidelines: 
+
+- Capitalize the first and last word of a headline
+- Capitalize words four letters and longer
+- Capitalize adjectives (orange, helpful), adverbs (quickly, easily), nouns (dog, truck), pronouns (she, our, themselves), verbs (run, connect), and subordinate conjunctions (once, as, if, when they connect unequal parts of a sentence)
```

Comment:
Add "is" and "are" to the verbs part, since many people put them in lowercase, not realizing these are verbs